internal/flag/flags.go

```diff
@@ -66,6 +66,7 @@ const (
 	IncludedNamespace               = "includedNamespace"               // IncludedNamespace flag
 	UsernameShort                   = "u"                               // UsernameShort flag
 	Password                        = "password"                        // Password flag
+	SkipMongoShellQuestions         = "skipMongoshQuestions"            // SkipMongoShellQuestions flag
```

Comment:
@gssbzn what do you think about the name of this flag?  I don’t like it but I am not sure how to call it

Reply:
I don't like it much either but wasn't going to nit pick given I lack a better alternative, some options (not necessarily better) just `skipMongosh`, `skipMongoshSetup`, `skipShellConnection`, `noMongosh`

Reply:
I like `skipMongosh`. Thanks!